examples/__main__.py

```diff
@@ -33,6 +44,8 @@ def __init__(self):
         self.codeBtn = QtGui.QPushButton('Run Edited Code')
         self.codeLayout = QtGui.QGridLayout()
         self.ui.codeView.setLayout(self.codeLayout)
+        #self.simulate_black_mode()
+        self.hl = PythonHighlighter(self.ui.codeView.document())
```

Comment:
To get highlighting to update with change of themes, add the following two lines after this line:

```python
        app = QtGui.QApplication.instance()
        app.paletteChanged.connect(self.updateTheme)
```

Also add the following method to the `ExampleLoader`

```
    def updateTheme(self):
        self.hl = PythonHighlighter(self.ui.codeView.document())
```

Also, please delete the commented line on line 47

Reply:
It's done, let me know if it works !